doc/docs/stdlib.html

```diff
@@ -618,3 +618,19 @@ <h4 id="mergePatch">std.mergePatch(target, patch)</h4>
 <p>Applies <code>patch</code> to <code>target</code> according to <a
 href="https://tools.ietf.org/html/rfc7396">RFC7396</a></p>
 
+<h3 id="Debugging">Debugging</h3>
+
+<h4 id="trace">std.trace(str, rest)</h4>
+
```

Comment:
I was hoping you could cut a release after this merges. Can I call it 0.10.1 ?

Reply:
@sparkprime decides about these things. 

It would probably be 0.11 since new features were added. We would also need to add documentation for #481 before. I'm all for frequent releases, but the current procedure for new releases is a bit tedious AFAIK.